larray/example.py

```diff
@@ -0,0 +1,43 @@
+import os
+import larray as la
+
+EXAMPLE_FILES_DIR = os.path.dirname(__file__) + '/tests/data/'
+
+def load_example_arrays():
```

Comment:
We should have an argument to this function because we will probably include several example datasets in the future. The goal was to have something akin to http://www.stata.com/help.cgi?sysuse. The question is whether pop+hh+qx is a single "Session" dataset, or if those are three individual LArray dataset which happen to be somewhat linked. I would lean towards the former, but then we need a name for the global thing :)